dll/win32/newdev/wizard.c

```diff
@@ -641,9 +641,13 @@ CHSourceDlgProc(
                 {
                     BROWSEINFO bi = { 0 };
                     LPITEMIDLIST pidl;
+                    HINSTANCE hInst = (HINSTANCE)GetWindowLong(hwndDlg, GWL_HINSTANCE);
```

Comment:
That should be GetWindowLongPtr()! (And maybe GWLP_ ? also)

Reply:
Thanks

Reply:
@tkreuzer  can you confirm this?

Reply:
This is correct (using the "Ptr" version) for portability of the code between 32 and 64 bits.